hs_app_netCDF/templates/pages/netcdfresource.html

```diff
@@ -126,14 +126,14 @@
     {% if cm.files.all %}
         {# tab body for ncdump #}
             {% for f in cm.files.all %}
-                {% if f.resource_file.name|slice:"-3:" == 'txt'%}
+                {% if f.short_path|slice:"-3:" == 'txt'%}
                     <div style="clear: both">
                         <br>
                         <legend>NetCDF Header Info</legend>
-                        <p>{{ f.resource_file.name|slice:"33:" }}</p>
+                        <p>{{ f.short_path }}</p>
                         <div class="hs-doc-preview">
-                            {% if f.resource_file.size <= 5000000 %}
-                                <pre id="netcdf-header-info">{{ f.resource_file.read}}</pre>
+                            {% if f.size <= 5000000 %}
+                                <pre id="netcdf-header-info">{{ f.read }}</pre>
```

Comment:
@alvacouch Never mind. I now see the new ```read``` property added in this PR. So it should work. Just ignore my comment above.